pkg/controller/instance/instance_controller_test.go

```diff
@@ -313,6 +313,13 @@ func TestInstanceController_Instance_Ready_Deletion(t *testing.T) {
 		t.Errorf("instance status unexpected: %s, expected: %s", fetched.Status.Phase.Name, registriesv1alpha1.InstanceStatusPhaseTerminating)
 	}
 
+	res, err = r.Reconcile(req)
```

Comment:
Can you maybe put a one-line comment here to explain, why you need to reconcile again here? Might help future people debugging this code.